bin/src/main.rs

```diff
@@ -261,7 +281,12 @@ async fn _main() {
 
     executor.init();
 
-    #[cfg(feature = "libjournald")]
+    // Use an internal env var to support running integration test w/o additional delays
+    let event_delay = std::env::var("LOGDNA_INTERNAL_FS_DELAY")
```

Comment:
hardcoded string needs to be replaced by a const from config::env_vars

Reply:
Good catch, there may be other things like this where the change predates the new convention, I missed this one